lib/AST/Module.cpp

```diff
@@ -507,19 +507,6 @@ void ModuleDecl::addFile(FileUnit &newFile) {
   clearLookupCache();
 }
 
-void ModuleDecl::removeFile(FileUnit &existingFile) {
-  // Do a reverse search; usually the file to be deleted will be at the end.
-  std::reverse_iterator<decltype(Files)::iterator> I(Files.end()),
-  E(Files.begin());
-  I = std::find(I, E, &existingFile);
-  assert(I != E);
-
-  // Adjust for the std::reverse_iterator offset.
-  ++I;
-  Files.erase(I.base());
-  clearLookupCache();
```

Comment:
The only remaining call to clearLookupCache() is from addFile(). Is that one still necessary?

Reply:
@slavapestov I *think* the Tensorflow folk are currently relying on it to clear the module-level cache for when they add a `SynthesizedFileUnit` to the module. I'm hoping to take care of that when I replace `addFile` with `setFiles`.

Reply:
Ah yeah, I just saw that in your PR description. Makes sense, thanks!